### PR TITLE
feat: introduce Travis CI config

### PR DESCRIPTION
**Pull request by shuplenkov**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+  - mkdir -p ~/.local/bin
+  - curl -fsSL -o ~/.local/bin/dash-network https://raw.github.com/dashpay/dash-network-deploy/master/bin/dash-network && chmod +x ~/.local/bin/dash-network
+  - git clone https://$NETWORK_CONFIGS_GITHUB_USERNAME:$NETWORK_CONFIGS_GITHUB_TOKEN@github.com/nmarley/dash-network-configs ~/dash-network-configs
+  - cp ~/dash-network-configs/$NETWORK_CONFIGS_PATH/.env.example ~/dash-network-configs/$NETWORK_CONFIGS_PATH/.env
+
+script:
+  - cd ~/dash-network-configs/$NETWORK_CONFIGS_PATH
+  - ~/.local/bin/dash-network test $DEVNET
+  - cd $TRAVIS_BUILD_DIR
+  - docker run dashpay/platform-test-suite $DAPI_SEED --faucet-key=$FAUCET_PRIVATE_KEY
```
Comment (shumkov) [inline .travis.yml]: Does it work already?

Comment (shuplenkov) [reply .travis.yml]: Kind of

https://travis-ci.com/github/dashevo/evonet-test-ci

Comment (shumkov) [reply .travis.yml]: It seems not https://travis-ci.com/github/dashevo/evonet-test-ci/jobs/328011587#L2299

Probably, we need to build the image before we merge it. Also, let's set the docker tag with an env variable in the travis build settings.

**Review by shumkov — APPROVED** [review]
Looks good!

**Pull request merged**